WordPress/build.gradle

```diff
@@ -153,24 +153,28 @@ android {
             buildConfigField "boolean", "ENABLE_FEATURE_CONFIGURATION", "false"
             buildConfigField "boolean", "CONTACT_INFO_BLOCK_AVAILABLE", "true"
             buildConfigField "boolean", "LAYOUT_GRID_BLOCK_AVAILABLE", "false"
+            buildConfigField "boolean", "ENABLE_EDITOR_HELP_OPTION", "false"
         }
 
         zalpha { // alpha version - enable experimental features
             dimension "buildType"
             buildConfigField "boolean", "VIDEO_OPTIMIZATION_AVAILABLE", "true"
             buildConfigField "boolean", "ENABLE_FEATURE_CONFIGURATION", "false"
+            buildConfigField "boolean", "ENABLE_EDITOR_HELP_OPTION", "true"
         }
 
         wasabi { // "hot" version, can be installed along release, alpha or beta versions
             applicationIdSuffix ".beta"
             dimension "buildType"
             buildConfigField "boolean", "UNIFIED_COMMENTS_LIST", "true"
+            buildConfigField "boolean", "ENABLE_EDITOR_HELP_OPTION", "true"
         }
 
         jalapeno { // Pre-Alpha version, used for PR builds, can be installed along release, alpha, beta, dev versions
             isDefault true
             applicationIdSuffix ".prealpha"
             dimension "buildType"
+            buildConfigField "boolean", "ENABLE_EDITOR_HELP_OPTION", "true"
```

Comment:
I originally added a new [build config value](https://github.com/wordpress-mobile/WordPress-Android/pull/14558/commits/93c069450f4016452089d40b0e1a20d973ce7b2a) for the editor onboarding features, to enable it only for develop and PR builds. However, after [some feedback](https://github.com/wordpress-mobile/WordPress-Android/pull/14558#discussion_r628252404) I opted to rely upon `Build.DEBUG` instead. WDYT about that alternative? Would that alternative accomplish what we need here?

Reply:
Good idea! Fixed in 3a7e06d02804d232bf21662a2c6acdbfff04174b